src/pages/events/ri3d/index.astro

```diff
@@ -180,13 +180,13 @@ const optimizedBanner = await getImage({
       <h1 class="display-2 p-2 fw-bold d-inline-flex">FAMNM Robot in 3 Days</h1>
       <div class="col mb-4">
         <a
-          href="https://youtube.com/playlist?list=PLSr939s0l0js6fvrbCpADueFlBfIQRw-v&si=94LVXDRi6rrx4gDr"
+          href="https://www.youtube.com/@FAMNM-Ri3D"
           class="btn btn-lg btn-primary"
           role="button"
           rel="noreferrer noopener"
           target="_blank"
         >
-          2024 Videos&nbsp;
+          2025 Videos&nbsp;
```

Comment:
For scalability and organization later on, a playlist for each season would be nice